Update index.mdx

tomuk5 opened the pull request:
Add T-Beam Supreme

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=649) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/meshtastic/meshtastic?pullRequest=649) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/meshtastic/meshtastic?pullRequest=649) it.</sub>

tomuk5 commented on docs/hardware/devices/tbeam/index.mdx:
```diff
@@ -155,5 +155,23 @@ This is an earlier version of the T-Beam board. Due to changes in the design thi
+  - 915 MHz
+  - 923 MHz
+- **Navigation Module**
+  - **NEO-M10S - GNSS receiver (supports GPS, GLONASS, Galileo, BeiDou)** (better GPS sensitivity)
```
Agreed, will add

tomuk5 replied:
added

caveman99 commented:
The s3 core module can work standalone and will run meshtastic just fine, in a headless configuration. I'd add an explanation to the listings but i think we can keep it as a separate option.

tomuk5 commented:
@caveman99 i've added the brief description for the S3 module headless operation now and added the firmware file naming convention too. Tabs should be up to the same spec as existing hardware tabs.

tomuk5 commented:
> 

are you now happy to review after reading the previous comment?
